frontend/screens/Friends.js

```diff
@@ -63,15 +63,19 @@ export default class Friends extends React.Component {
     this.setState({ friends: newData })
   }
 
-  async removeRequest(friend, newArr) {
-    friendsApi
-      .removeFriendship(friend)
-      .then(() => {
-        this.setState({ friends: newArr })
-      })
-      .catch(() => {
-        this.setState({ errorAlert: true })
-      })
+  async removeRequest(friend, newArr, status) {
```

Comment:
is status needed? i saw when the function was being invoked it didnt take in status